cars/car.js

```diff
@@ -5,5 +5,22 @@ function Car(mpg) {
 }
```

Comment:
It looks like the method signature needs to be shaped like: `Car(mpg)` - and the `this.mpg = mpg` needs to be set in the constructor function.